File: ControlRemote/Web/Dto/EmployerCreateModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Web.Dto
{
    public class EmployerCreateModel
    {
        public int ManagerId { get; set; }
        public string Name { get; set; }
        public string Login { get; set; }
    }
}

File: ControlRemote/Web/ClientApp/src/app/models/EmployerCreateModel.ts
export class EmployerCreateModel {
    managerId: number;
    name: string;
    login: string;

    public constructor(_managerId: number, _name: string, _login: string) {
        this.managerId = _managerId;
        this.name = _name;
        this.login = _login;
    }
}